### ISSUE TITLE
[feature reqest] adjust time 1-2 seconds earlier

### ISSUE THREAD

**Issue by GoogleCodeExporter** [quemoni/subtitleedit#193]
```
For a text where the time displayed is shorter than it should, it would be 
great subtitles can adjust 1-2 seconds earlier as well as seconds later like 
what is now.

1- The software would try to adjust with seconds later until the max it could
2- The software would try to continue with adjusting with 1-2 seconds max 
earlier. (too much would not be wanting)

Thanks!
```

Original issue reported on code.google.com by `danielgi...@gmail.com` on 3 Jan 2014 at 10:15

**Comment by GoogleCodeExporter** [comment]
```
Hi Daniel,

Should already be fixed in revision: r2335 (allows up to 1.5 seconds earlier)
http://www.nikse.dk/SubtitleEdit.zip (latest portable beta), will be included 
in SE 3.3.12 final
```

Original comment by `nikse.dk@gmail.com` on 3 Jan 2014 at 11:59
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hi.
With this last beta version:
There are a couple of places where the software was not able to fix short
display time by taking time before and after. Note: I set up the min.
duration for a line from 500ms to 800ms.

"Unable to fix text number..."

Is this new feature is taking account of the min. duration option?
A printscreen included.

Have a nice weekend, Lord of the Programmers!

Daniel
```

Original comment by `danielgi...@gmail.com` on 4 Jan 2014 at 3:47
